docs/content/en/tasks/backup-restore/azure/_index.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Backup and restore with Azure Storage"
+linkTitle: "Azure Blob Storage"
+no_list: true
+weight: 3
+description: Use Medusa to backup and restore Apache Cassandra® data in Kubernetes to Azure Storage.
+---
+
+Medusa is a Cassandra backup and restore tool. It's packaged with K8ssandra and supports a variety of backends,
+including Azure Storage.
+
+## Introduction
+
+Azure Storage is a RESTful online file storage service for storing and accessing data on Microsoft Azure / AKS
+infrastructure.
+
+For details about storing blobs on Azure, read the [Azure Blob Storage
+documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction).
+
+## Create a storage account for backups
+
+In order to perform backups in Azure, Medusa needs to use a storage account with appropriate permissions.
+
+Using the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/), run the following command to create the
+`medusabackups` storage account:
+
+```bash
+az storage account create \
+  --name medusabackups \
+  --resource-group storage-resource-group \
+  --location eastus \
+  --sku Standard_RAGRS \
+  --kind StorageV2
+```
+
+See [this article](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create) for details on storage
+account creation on Azure.
+
+## Create the storage account credentials file
+
+List the access keys that were generated for the storage account:
+
+```bash
+az storage account keys list \
+    --account-name medusabackups \
+    --resource-group storage-resource-group
+```
+
+Create a file on your local machine called `credentials.json` with the following contents:
```

Comment:
I wonder if we should simplify the contents of the secret.

* It could contain just the access key instead of a JSON doc containing the key and the account name;
* The storage account name could go in the K8ssandra chart under `medusa.storage_properties`.

Thoughts?

Reply:
Hmm scratch that, probably not feasible without changing Medusa itself since it expects that JSON file with those exact contents.

Reply:
Actually that might still be possible: K8ssandra could generate a secret definition containing the JSON data as expected by Medusa. I will work on that today and see how it goes, but I think it could be a nice usability improvement if we could skip the manual generation of that JSON file.

Reply:
As decided offline, we are going to keep things as they are for now and revisit this later for K8ssandra 2.0.